src/reducers/notebook-reducer.js

```diff
@@ -182,6 +183,11 @@ const notebookReducer = (state = newNotebook(), action) => {
 
     case 'ADD_LANGUAGE_TO_EDITOR': {
       const { languageDefinition } = action
+      addChangeLanguageTask(
+        languageDefinition.languageId,
+        languageDefinition.displayName,
+        languageDefinition.keybinding,
+      )
```

Comment:
this should happen in the addLanguageToEditor action creator rather than the reducer -- we want to move toward the reducer being a pure function of (state, action) with zero side effects. (when @hamilton and I were starting out with Redux, we didn't know that this was the best practice -- that it would make testing easier etc. But now that we're using thunk, we can move in that direction)

Reply:
oops, that action creator is actually called `addLanguage` i think.